react/features/mobile/callkit/middleware.js

```diff
@@ -230,10 +230,13 @@ function _conferenceWillJoin({ getState }, next, action) {
         .then(() => {
             const { room } = state['features/base/conference'];
             const { callee } = state['features/base/jwt'];
+            const { muted } = state['features/base/media'].audio;
```

Comment:
We're saying that CallKit is to depict our desire/intention to have the audio muted? Why are we not saying that CallKit should depict reality? Because we're in CONFERENCE_WILL_JOIN?

Reply:
This is the initial moment in which we "invoke" CallKit for the first time. As such, we need to tell it the initial muted state (for the CallKit UI). What we do here is tell CallKit the real muted state. After this call, the muted state will be synced with the SET_AUDIO_MUTED action (this is already inplace).